common/src/main/java/com/wynntils/models/stats/type/ShinyStatType.java

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.stats.type;
+
+public record ShinyStatType(int id, String key, String displayName, StatUnit statUnit) {
+    public static final ShinyStatType UNKNOWN = new ShinyStatType(0, "unknown", "Unknown", StatUnit.RAW);
```

Comment:
Does the count of real shiny stats start at 1?

Reply:
Yes.